packages/contracts/abis/IEndpoint.json

```diff
@@ -1,298 +1,97 @@
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "uint32",
-        "name": "healthGroup",
-        "type": "uint32"
-      }
-    ],
-    "name": "getPricesX18",
-    "outputs": [
-      {
-        "components": [
-          {
-            "internalType": "int128",
-            "name": "spotPriceX18",
-            "type": "int128"
-          },
-          {
-            "internalType": "int128",
-            "name": "perpPriceX18",
-            "type": "int128"
-          }
-        ],
-        "internalType": "struct IEndpoint.Prices",
-        "name": "",
-        "type": "tuple"
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "uint64",
-        "name": "subaccountId",
-        "type": "uint64"
-      }
-    ],
-    "name": "getSubaccountById",
-    "outputs": [
-      {
-        "internalType": "bytes32",
-        "name": "",
-        "type": "bytes32"
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "bytes32",
-        "name": "subaccount",
-        "type": "bytes32"
-      }
-    ],
-    "name": "getSubaccountId",
-    "outputs": [
-      {
-        "internalType": "uint64",
-        "name": "",
-        "type": "uint64"
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
-  },
-  {
-    "inputs": [],
-    "name": "getTime",
-    "outputs": [
-      {
-        "internalType": "uint128",
-        "name": "",
-        "type": "uint128"
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
-  },
-  {
-    "inputs": [],
-    "name": "getVersion",
-    "outputs": [
-      {
-        "internalType": "uint64",
-        "name": "",
-        "type": "uint64"
-      }
-    ],
-    "stateMutability": "nonpayable",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "uint32",
-        "name": "productId",
-        "type": "uint32"
-      },
-      {
-        "internalType": "address",
-        "name": "book",
-        "type": "address"
-      }
-    ],
-    "name": "setBook",
-    "outputs": [],
-    "stateMutability": "nonpayable",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "bytes",
-        "name": "transaction",
-        "type": "bytes"
-      }
-    ],
-    "name": "submitSlowModeTransaction",
-    "outputs": [],
-    "stateMutability": "nonpayable",
-    "type": "function"
-  },
-  {
-    "inputs": [
-      {
-        "internalType": "uint64",
-        "name": "idx",
-        "type": "uint64"
-      },
-      {
-        "internalType": "bytes[]",
-        "name": "transactions",
-        "type": "bytes[]"
-      }
-    ],
-    "name": "submitTransactionsChecked",
-    "outputs": [],
-    "stateMutability": "nonpayable",
-    "type": "function"
-  }
-]
+{
```

Comment:
this is wrong -- it seems to be replacing the correct IEndpoint.json (from vertex-core) with the stubbed out IEndpoint from the LBA -- my guess is we don't want the LBA endpoint interface.